src/framework/theme/components/dialog/dialog.service.ts

```diff
@@ -150,7 +150,8 @@ export class NbDialogService {
   /**
    * Opens new instance of the dialog, may receive optional config.
    * */
-  open<T>(content: Type<T> | TemplateRef<T>, userConfig: Partial<NbDialogConfig<T>> = {}): NbDialogRef<T> {
+  open<T>(content: Type<T> | TemplateRef<T>,
+          userConfig: Partial<NbDialogConfig<Partial<T> | string>> = {}): NbDialogRef<T> {
```

Comment:
why do we need `string` here?

Reply:
My First commit did not have string. It gave me compilation error in [Travis build](https://travis-ci.org/akveo/nebular/jobs/482766836) for below source file. 

https://github.com/akveo/nebular/blob/37eaea73db8eb45ffa2d352c59028a0f507c1c8f/src/playground/with-layout/dialog/dialog-template.component.ts#L30